src/structures/Message.js

```diff
@@ -133,6 +134,17 @@ class Message extends Base {
       }
     }
 
+    /**
+     * A collection of stickers in the message
+     * @type {Collection<Snowflake, Sticker>}
+     */
+    this.stickers = new Collection();
+    if (data.stickers) {
+      for (const sticker of data.stickers) {
+        this.stickers.set(sticker.id, new Sticker(this.client, sticker));
+      }
+    }
```

Comment:
```suggestion
    this.stickers = new Collection(data.stickers?.map(sticker => [sticker.id, new Sticker(this.client, sticker)]));
```

Reply:
this suggestion might be better, but wouldn't it be more consistent to leave it as it is now, as nowhere else is a collection actually constructed like this, its usually using `reduce` or a for loop

Reply:
I also agree with NotSugden. I think it looks better with for-of.